src/gui/application.cpp

```diff
@@ -1016,9 +1017,10 @@ void Application::setupTranslations()
         qCWarning(lcApplication()) << trPath << "folder containing translations is missing. Impossible to load translations";
         return;
     }
-    const QString trFile = QLatin1String("client_") + lang;
+    const QString trFile = QLatin1String("client");
+    const auto trPrefix = QLatin1String("_");
     qCDebug(lcApplication()) << "trying to load" << lang << "in" << trFile << "from" << trPath;
-    if (translator->load(trFile, trPath) || lang.startsWith(QLatin1String("en"))) {
+    if (translator->load(systemLocale, trFile, trPrefix, trPath) || lang.startsWith(QLatin1String("en"))) {
         // Permissive approach: Qt and keychain translations
         // may be missing, but Qt translations must be there in order
         // for us to accept the language. Otherwise, we try with the next.
```

Comment:
I think this comment is obsolete -- there is no next language to try.

`QTranslator::load(const QLocale&, ...)` will do this on its own